### PR TITLE
docs: use latest cloud-rad version

### PR DESCRIPTION
**Pull request by dansaadati**
This version update will use the refactored
nodejs-cloud-rad generator, and pull in new versions.

### CODE REVIEW

**Comment by sofisl** [comment]
@dansaadati, have you tried publishing with this branch? Just want to make sure it works before merging!

**Review by alexander-fenster — APPROVED** [review]

**Comment by dansaadati** [comment]
Holding off merging until logs are output: https://github.com/googleapis/nodejs-cloud-rad/pull/74 and new version successfully runs.

**Pull request merged**
